config/samples/velero_v1_serverstatusrequest.yaml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Something I missed in the BSL review was that we got this `samples` directory - is it used for anything other than documentation? We have the `site/content/docs/<version>/api-types/` directory for our other CRD docs, so I'm wondering if we could maybe take these contents and move them over post-generation.

However, I'm also not going to hold up this PR for it.

Reply:
Good point. These have no function other than to server as an example, and it's generated by kubebuilder (empty) so it's sorta expected (but again, not needed). Let's punt on this for now.